server/src/services/media.service.ts

```diff
@@ -70,7 +70,7 @@ export class MediaService {
   async handleQueueGenerateThumbnails({ force }: IBaseJob): Promise<JobStatus> {
     const assetPagination = usePagination(JOBS_ASSET_PAGINATION_SIZE, (pagination) => {
       return force
-        ? this.assetRepository.getAll(pagination, { isVisible: true, withDeleted: true })
```

Comment:
Do you think trashed assets should still get queued? Presumably the user doesn't care about them since they're trashed.

Reply:
I think it does make sense, as if a user later restores an asset from trash it could be confusing if the thumbnail for that is broken. Or even just looking at assets in the trash for that purpose

Reply:
Oh that's true, it could be good if they need to figure out what should be restored from the trash and the thumbnail is broken.

Reply:
Yes, I think so. You could argue that they'll likely be permanently deleted, which might seem wasteful. However, if they are restored you'll end up with assets that aren't in the correct format.